src/transformations/transform-constructor.ts

```diff
@@ -50,6 +50,9 @@ function getUnchainedArguments(
   }
 }
 
+// Removes the constructor keyword and parameters until the first `{`, for example
```

Comment:
Also mention that it is meant to be run after `transformConstructor`. And in the other function mention that this one has to run to clean up after.